backend/requirements.txt

```diff
@@ -0,0 +1,6 @@
+Django==2.2.13
+django-cors-headers
+pymongo
+dnspython
+gunicorn
+python-dotenv
```

Comment:
Please sort these and add specific versions (this will be better for the prod build.  Alternatively, freeze versions in a separate `requirements.prod.txt` (or the like)